service/switchboardapp/pom.xml

```diff
@@ -55,6 +58,19 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+
+		<dependency>
+			<groupId>org.openapitools</groupId>
+			<artifactId>openapi-generator</artifactId>
+			<version>${openapi-generator-version}</version>
+		</dependency>
+
+		<!--TODO remove this dependency when org.openapitools.openapi-generator releases version 5.0.0-->
+		<dependency>
+			<groupId>org.openapitools</groupId>
+			<artifactId>jackson-databind-nullable</artifactId>
+			<version>${openapi-jackson-nullable-version}</version>
+		</dependency>
```

Comment:
Before anyone asks about this, version 4.3.1 of the open api generator adds an usused import for this library in the models. It is a known bug which is tracked here https://github.com/OpenAPITools/openapi-generator/issues/5690. I personally tested with version 5.0.0-beta2 and the bug is fixed in that version, but I'd rather not depend on a beta version of a library. Another option would be to downgrade to an earlier version where the bug does not exist, but then we risk missing more recent features/bug fixes.